r/version_5/dm01_resave_spss_as_qs_v5.R

```diff
@@ -12,26 +12,31 @@ source('r/version_5/functions/save_spss_qs_v5.R')
 
 # Countries ---------------------------------------------------------------
 # in case running for certain countries only
-country_codes <- "Group1"
+args <- commandArgs(trailingOnly=TRUE)
+print(args)
+if (!exists("groups")) groups <- "Group1"
+if(length(args) == 1) groups <- args
 
 # Open SPSS and save as QS ------------------------------------------------
 
-for(country in country_codes){
-  print(paste0("Start: ", country))
-  filenames <- readxl::read_excel('data/spss_files.xlsx', sheet = country)
+for(group in groups){
```

Comment:
Is this needed anymore? I guess it can be deleted?

Reply:
yeah can be, meant to and then forgot haha